devices/ubisys/h1_v1.7.json

```diff
@@ -0,0 +1,322 @@
+          "public": false,
+          "refresh.interval": 3600,
+          "default": 14400
+        },
+        {
+          "name": "config/group",
+          "default": "auto"
+        },
+        {
+          "name": "config/heatsetpoint",
+          "refresh.interval": 3660,
+          "default": 0
+        },
+        {
+          "name": "config/mode",
+          "refresh.interval": 3660,
+          "read": {
+            "at": "0x001C",
+            "cl": "0x0201",
+            "ep": 1,
+            "fn": "zcl:attr"
+          },
+          "write": {
+            "at": "0x001C",
+            "cl": "0x0201",
+            "dt": "0x10",
+            "ep": 1,
+            "eval": "if (Item.val == 'off') { 0 } else if (Item.val == 'auto') { 1 } else if (Item.val == 'heat') { 4 };",
+            "fn": "zcl:attr"
+          },
+          "parse": {
+            "at": "0x001C",
+            "cl": "0x0201",
+            "ep": 1,
+            "eval": "if (Attr.val == 0) { Item.val = 'off' } else if (Attr.val == 1) { Item.val = 'auto' } else if (Attr.val == 4) { Item.val = 'heat' };",
+            "fn": "zcl:attr"
+          },
+          "default": "heat"
+        },
+        {
+          "name": "config/offset",
+          "refresh.interval": 3660,
+          "read": {
+            "at": "0x0003",
+            "cl": "0x0201",
+            "ep": 1,
+            "fn": "zcl:attr",
+            "mf": "0x10f2"
+          },
+          "write": {
+            "at": "0x0003",
+            "cl": "0x0201",
+            "dt": "0x30",
+            "ep": 1,
+            "eval": "parseInt(Item.val / 100);",
+            "fn": "zcl:attr",
+            "mf": "0x10f2"
+          },
+          "parse": {
+            "at": "0x0003",
+            "cl": "0x0201",
+            "ep": 1,
+            "eval": "Item.val = Attr.val * 100;",
+            "fn": "zcl:attr",
+            "mf": "0x10f2"
+          },
+          "default": 0
+        },
+        {
+          "name": "config/on"
+        },
+        {
+          "name": "config/reachable"
+        },
+        {
+          "name": "config/schedule",
+          "refresh.interval": 3660,
+          "read": {
+            "cl": "0x0201",
+            "cmd": "0x02",
+            "ep": "0x01",
+            "eval": "'7F01'",
+            "fn": "zcl:cmd"
+          }
+        },
+        {
+          "name": "config/schedule_on",
+          "static": true
+        },
+        {
+          "name": "config/unoccupiedheatsetpoint",
+          "refresh.interval": 3660,
+          "default": 1600
+        },
+        {
+          "name": "config/windowopendetectionenabled",
+          "read": {
+            "at": "0x0015",
+            "cl": "0x0201",
+            "ep": 1,
+            "mf": "0x10f2"
+          },
+          "write": {
+            "at": "0x0015",
+            "cl": "0x0201",
+            "dt": "0x10",
+            "ep": 1,
+            "eval": "Item.val",
+            "mf": "0x10f2"
```

Comment:
dt is 0x18 (bitmap) here. Eval should be something like the following, just to be on the safe side, assuming 1 means enabled and 0 disabled:
`if (Item.val) { 1 } else { 0 }`

And please add the `fn` parameter within the functions.

Reply:
I did it and tried again:

```
[
  {
    "success": {
      "/sensors/122/config/windowopendetectionenabled": true
    }
  }
]
```

it's true for a short time but then false again

`    "windowopendetectionenabled": false`

Enabled it via display and it works immediately

`    "windowopendetectionenabled": true`